integration-tests/__tests__/coverage_remapping.test.js

```diff
@@ -11,15 +11,12 @@
 
 const {readFileSync} = require('fs');
 const path = require('path');
-const SkipOnWindows = require('../../scripts/SkipOnWindows');
 const {cleanup, run} = require('../utils');
 const runJest = require('../runJest');
 
 const dir = path.resolve(__dirname, '../coverage-remapping');
 const coverageDir = path.join(dir, 'coverage');
 
-SkipOnWindows.suite();
```

Comment:
So this is the only test that spawns different results on windows - it adds `covered-test.ts` next to `covered.ts`. Ideas what can cause that? cc @SimenB @jwbay @Aftabnack

Reply:
https://github.com/facebook/jest/blob/2df9e4cc4b93abc71659c216c07d1a6c78909f0b/packages/jest-runner/src/run_test.js#L126-L127

* Here it gets the coverage and soucemap info from the from the runner, coverage will be there only for files being used in the tests.
* But soucemaps, along with containing maps for tested code, it also contains mapping for test file itself.

https://github.com/facebook/jest/blob/2df9e4cc4b93abc71659c216c07d1a6c78909f0b/packages/jest-cli/src/reporters/coverage_reporter.js#L55-L77

* Here it iterates over sourcemaps and tries to add it to the coverage map.

My guess, this addition works for test file itself, which it doesn't in any other testcase. Do we have another testcase which is written in `ts`, same behaviour should be seen